src/api/java/openiss/ws/soap/service/OpenISSSOAPServiceImpl.java

```diff
@@ -0,0 +1,96 @@
+package api.java.openiss.ws.soap.service;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UnsupportedEncodingException;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.ProtocolException;
+import java.net.URL;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.jws.WebService;
+import org.opencv.core.CvType;
+import org.opencv.core.Mat;
+import org.opencv.core.Core;
+import org.opencv.imgcodecs.Imgcodecs;
+import org.opencv.imgproc.Imgproc;
+
+/**
+ * Service Implementation
+ *
+ * @author zephi
+ */
+@WebService(endpointInterface = "api.java.openiss.ws.soap.service.OpenISSSOAPService")
+public class OpenISSSOAPServiceImpl implements OpenISSSOAPService {
+
+    @Override
+    public String fetchXML(String uri, String reqType) {
```

Comment:
(D) please remove all this craft; it does not belong to OpenISS

Reply:
checked, (D) all crafts have been removed.

Reply:
by the way, there are two (D) reviews this one and the one above.